pkg/registry/apis/iam/models.go

```diff
@@ -0,0 +1,44 @@
+package iam
+
+import (
+	"github.com/grafana/authlib/types"
+	"github.com/grafana/grafana/pkg/registry/apis/iam/legacy"
+	"github.com/grafana/grafana/pkg/registry/apis/iam/user"
+	"github.com/grafana/grafana/pkg/services/apiserver/builder"
+	"github.com/grafana/grafana/pkg/services/ssosettings"
+	"github.com/grafana/grafana/pkg/storage/unified/resource"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+var _ builder.APIGroupBuilder = (*IdentityAccessManagementAPIBuilder)(nil)
+var _ builder.APIGroupRouteProvider = (*IdentityAccessManagementAPIBuilder)(nil)
+
+// CoreRoleStorageBackend uses the resource.StorageBackend interface to provide storage for core roles.
+// Used wire to identify the storage backend for core roles.
+type CoreRoleStorageBackend interface{ resource.StorageBackend }
```

Comment:
Are CoreRoles always fixed?  do they actually depend on reading a database or not?

If not -- this could be a read only resource 🤔